x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -174,6 +176,9 @@ export const policyFactory = (
       },
     },
   };
+  policy.meta.billable = isBillablePolicy(policy);
```

Comment:
why don't you add this value in line 32 ?

Reply:
This value is calculated based on the rest of the policy so we need the entire policy first.